src/includes.ts

```diff
@@ -0,0 +1,2 @@
+export const includes = <T>(array: Array<T>, searchElement: T): boolean =>
+  array.findIndex((element) => element === searchElement) !== -1
```

Comment:
indexOf instead of findIndex, it handle the equality and avoid the lambda creation